tests/StackExchange.Redis.Tests/Sets.cs

```diff
@@ -343,4 +343,42 @@ public void SetPopMulti_Nil()
         var arr = db.SetPop(key, 1);
         Assert.Empty(arr);
     }
+
+    [Fact]
+    public async Task TestSort()
+    {
+        using var conn = Create();
+
+        var db = conn.GetDatabase();
+        var key = Me();
+        await db.KeyDeleteAsync(key);
+
+        var random = new Random();
+        var items = Enumerable.Repeat(0, 200).Select(x => random.Next()).ToList();
+        await db.SetAddAsync(key, items.Select(x=>(RedisValue)x).ToArray());
+        items.Sort();
+
+        var result = (await db.SortAsync(key)).Select(x=>(int)x);
+        Assert.Equal(items.Count,result.Count() );
+        Assert.Equivalent(items, result);
```

Comment:
There's an `IEnumerable<T>` overload to `Assert.Equal()` we'd want to use here:
```suggestion
        Assert.Equal(items, result);
```